views/edit_or_add_mixin.pug

```diff
@@ -43,10 +45,11 @@ mixin children_add(thisChoral)
         select#childChoral.form-control.add-child-input(name='childChoral')
           option(value="") No Children Added
           each choral in chorals
-            -if(!thisChoral || (choral.choralId != thisChoral.choralId))
+            -if((!thisChoral || (choral.choralId != thisChoral.choralId)) && (!children || !(children.find((c) => c.choralId == choral.choralId ))))
```

Comment:
I don't think this is a good idea it means that if you accidentally remove a child you can't read it. Better to do this all in jQuery